skrifa/src/patchmap.rs

```diff
@@ -293,6 +434,44 @@ impl PatchUri {
     }
 }
 
+/// Stores a materialized version of an IFT patchmap entry.
+///
+/// See: <https://w3c.github.io/IFT/Overview.html#patch-map-dfn>
+#[derive(Debug, Clone, Eq, PartialEq, Hash)]
+struct Entry {
+    // Key
+    codepoints: IntSet<u32>,
+    feature_tags: BTreeSet<Tag>,
+    design_space: Vec<DesignSpaceSegment>,
+
+    // Value
+    uri: PatchUri,
+    compatibility_id: [u32; 4], // TODO: Make this a reference?
```

Comment:
this feels lower value, a reference is only one machine word less than copying this around.

Reply:
Good point.